peekaboo/ruleset/expressions.py

```diff
@@ -205,7 +207,7 @@ class EvalRegexIterableMixIn(object):
     """ Common functionality for iterables which may contain regular
     expressions. """
     def __init__(self, tokens):
-        super().__init__(tokens)
+        super(EvalRegexIterableMixIn, self).__init__(tokens)
```

Comment:
Does your git-lint use pylint from python2?

Reply:
```
pylint --version
pylint 2.3.1
astroid 2.2.5
Python 3.7.4 (default, Jul 16 2019, 07:12:58) 
[GCC 9.1.0]
```

Reply:
For me, pylint3 (python 3.6 on Ubuntu 18.04) does not comment on the python3 `super().__init__` form but pylint (python2) does. OTOH I get a warning from pylint3 that `KeyError` has no member `message` (`e.args[0]` appears to be the generic solution there). Perhaps your git-lint is in a venv with python2 as interpreter and you checked the host pylint's version?

I'd prefer the python3 notation of `super()` because it's much cleaner.

Also I'm not sure but think that putting `future` imports at the very top is recommended and was therefore intentional.